src/live-preview.ts

```diff
@@ -299,19 +308,25 @@ export default class LivePreview {
         const cslpTag = this.tooltip.getAttribute("current-data-cslp");
 
         if (cslpTag) {
-            const [content_type_uid, entry_uid, locale, ...field] =
-                cslpTag.split(".");
-
+            const {
+                content_type_uid,
+                entry_uid,
+                locale,
+                variant,
+                fieldPath: field,
+            } = extractDetailsFromCslp(cslpTag);
+            console.log("🚀 ~ LivePreview ~ scrollHandler ~ variant:", variant);
```

Comment:
Remove this console log @contentstackMridul

Reply:
done